Update index.html - Übersetzung head, sidebar &footer

herrhundt opened the pull request:
Natürlich muss die words noch eingefügt werden, sobald diese fertig reviewed ist.

herrhundt committed:
Update index.html - Übersetzung head, sidebar &footer 

Natürlich muss die words noch eingefügt werden, sobald diese fertig reviewed ist.

herrhundt committed:
Update index.html

Credits im footer ergänzt

herrhundt committed:
Update index.html - footer ergänzt

Credits im footer ergänzt - bitte schließt euch hier an.

lenaschimmel commented on index.html:
```diff
@@ -4,25 +4,25 @@
 
 	<!-- THE BIG META -->
-	<title>Wie geht es weiter? CoViD-19-Verläufe durch interaktive Simulationen erklärt</title>  <!-- TRANSLATE -->
+	<title>Wie geht's weiter? Zukunftsszenarien COVID-19-Simulationsspiel</title>  <!-- TRANSLATE -->
```
Die Formulierung und Zeichensetzung "Zukunftsszenarien von COVID-19: ein Simulationsspiel" aus #38 klingt für mich deutlich griffiger.

herrhundt committed:
Update index.html - Verbesserungsvorschläge

eBildungslabor committed:
Update index.html

Credit ergänzt

lenaschimmel commented on index.html:
```diff
@@ -316,22 +322,22 @@
 		der Bilder, der Quelltexte und der Texte auf dieser Seite - in Blogs, auf Nachrichtenseiten, in der Schule, überall!
 		<a href='https://github.com/ncase/covid-19'>
-			(den Quelltext finden sie auf Github)
+			(den Quelltext findest du auf Github)
```
Ich glaube, das Repo der deutschen Übersetzung (also dieses hier) ist noch nirgends sonst verlinkt, oder? Dann wäre hier vielleicht der richtige Ort, ggf.:

> (den Quelltext [dieser Seite](https://github.com/TQueV/covid-19) bzw. [des englischen Originals](https://github.com/ncase/covid-19) findet du auf GitHub)

lenaschimmel replied:
Vielleich würden Menschen, die später noch etwas an dieser Übersetzung verbessern möchten, auch [oben](https://github.com/TQueV/covid-19/pull/40/files#diff-eacf331f0ffc35d4b482f1d15a887d3bR48) bei dem "Willst du helfen?" suchen, wo auch nur das englische Repo verlinkt ist?

jkoelling replied:
Ich würde an beiden Stellen auf das Original-Repo und die Übersetzung verweisen.

jkoelling commented:
Ich hab im Chat mal ne Diskussion für "playable simulation" (hier meist als "Simulationsspiel" übersetzt) eröffnet. Da das an mehreren Stellen in der Übersetzung vorkommt und aktuell nicht alle an den PRs beteiligt sind, hoffe ich da mehr Feedback einsammeln zu können.

jkoelling commented on index.html:
```diff
@@ -343,16 +349,16 @@
 
 		Diese kostenlose Open-Source-Anleitung wurde ermöglicht dank
```
wir hatten hier an anderen Stellen was anderes als "Anleitung" für "guide" gewählt, würde da konsistent bleiben.

herrhundt replied:
Bin jetzt an mehreren Stellen auf "Führer" gestoßen - hatten wir das wirklich?

jkoelling commented on index.html:
```diff
@@ -343,16 +349,16 @@
 		Diese kostenlose Open-Source-Anleitung wurde ermöglicht dank
 		Nickys Unterstützer auf Patreon.
-		Wenn sie (und nur dann) in diesen schweren Zeiten über ein ausreichendes Einkommen verfügen, können Sie
+		Wenn du (und nur dann) in diesen schweren Zeiten über ein ausreichendes Einkommen verfügst, kannst du
```
```suggestion
		Wenn du in diesen schweren Zeiten über ein mehr als ausreichendes Einkommen verfügst (und wirklich NUR DANN), kannst du
```
näher am Ton des Originals, finde ich hier besonders wichtig da es ja um einen Aufruf für Spenden direkt an Nicky geht

jkoelling committed:
fixes humor

Original war: "(excellent for homeshooling if, say, schools are currently closed due to a pandemic)"
Die erste Übersetzung hatte den tongue-in-cheek Aspekt komplett entfernt,

herrhundt committed:
Update index.html - spielbare Simulation

herrhundt committed:
Update index.html

Co-authored-by: Jan Kölling <jkoelling@users.noreply.github.com>

herrhundt committed:
Update index.html
